ecs.tf

```diff
@@ -115,6 +115,63 @@ variable "ecsInstancerolePolicy" {
 EOF
 }
 
+variable "use_ssm" {
+  default     = false
+  description = "Allow access via AWS Systems Mamager"
+}
+
+variable "ecsInstanceroleSsmPolicy" {
+  type = string
+
+  default = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ecs:CreateCluster",
+        "ecs:DeregisterContainerInstance",
+        "ecs:DiscoverPollEndpoint",
+        "ecs:Poll",
+        "ecs:RegisterContainerInstance",
+        "ecs:StartTelemetrySession",
+        "ecs:Submit*",
+        "ecr:GetAuthorizationToken",
+        "ecr:BatchCheckLayerAvailability",
+        "ecr:GetDownloadUrlForLayer",
+        "ecr:BatchGetImage",
+        "logs:CreateLogStream",
+        "logs:PutLogEvents"
+      ],
+      "Resource": "*"
```

Comment:
Is is possible, without too much difficulty, to make the `Resource` on this and the other two rules more specific?

Reply:
I agree that doing that seems like it would be a good improvement